nlmod/sim/sim.py

```diff
@@ -64,7 +64,7 @@ def write_and_run(sim, ds, write_ds=True, script_path=None, silent=False):
     ds.attrs["model_ran_on"] = dt.datetime.now().strftime("%Y%m%d_%H:%M:%S")
 
 
-def get_tdis_perioddata(ds):
+def get_tdis_perioddata(ds, nstp=None, tsmult=None):
```

Comment:
I prefer not to have `nstp` and `tsmult` as keyword arguments here because than you can end up with different settings in your dataset and your tdis.